clr-k8s-examples/tests/autoscale/load-gen.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Pod
```

Comment:
@krsna1729 should we make this a deployment. That way we can scale it up to generate more load?